test/helpers/kubectl.go

```diff
@@ -912,37 +911,87 @@ func (kub *Kubectl) CiliumReport(namespace string, commands ...string) {
 	kub.GatherLogs()
 }
 
+// CiliumCheckReport prints a few checks on the Junit output to provide more context to users.
+func (kub *Kubectl) CiliumCheckReport() {
+	pods, _ := kub.GetCiliumPods(KubeSystemNamespace)
+	fmt.Fprintf(CheckLogs, "Cilium pods: %v\n", pods)
+
+	var policiesFilter = `{range .items[*]}{.metadata.namespace}{"::"}{.metadata.name}{" "}{end}`
+	netpols := kub.Exec(fmt.Sprintf(
+		"%s get netpol -o jsonpath='%s' --all-namespaces",
+		KubectlCmd, policiesFilter))
+	fmt.Fprintf(CheckLogs, "Netpols loaded: %v\n", netpols.Output())
+
+	cnp := kub.Exec(fmt.Sprintf(
+		"%s get cnp -o jsonpath='%s' --all-namespaces",
+		KubectlCmd, policiesFilter))
+	fmt.Fprintf(CheckLogs, "CiliumNetworkPolicies loaded: %v\n", cnp.Output())
+
+	cepFilter := `{range .items[*]}{.metadata.name}{"="}{.status.status.policy.realized.policy-enabled}{"\n"}{end}`
+	cepStatus := kub.Exec(fmt.Sprintf(
+		"%s get cep -o jsonpath='%s' --all-namespaces",
+		KubectlCmd, cepFilter))
+
+	fmt.Fprintf(CheckLogs, "Cilium CEP status:\n")
+	for pod, policy := range cepStatus.KVOutput() {
+		fmt.Fprintf(CheckLogs, "\t%s=> %s\n", pod, policy)
```

Comment:
It might also be better to print these with a tabwriter(https://golang.org/pkg/text/tabwriter/)  so they align better.